Detectors/EMCAL/reconstruction/include/EMCALReconstruction/RAWDataHeader.h

```diff
@@ -0,0 +1,74 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
```

Comment:
No, this is just a temporary structure which we need for the moment because the we still send a mixed header (CDH from SRU updated in words2 and 3 by ReadoutCard). Once the SRU firmware is adapted to the RDHv4 I will remove this header and replace it in the decoder using the header from DataFormats/Headers/RAWDataHeader.h. But for lab tests we with the current SRU we need to port our own header for a while.